File: app/src/main/java/com/example/kalkulatorbmi/MainActivity.java
package com.example.kalkulatorbmi;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
    public void FirstScreen(View view) {
        Intent first = new Intent(this, SecondeActivity.class);
        startActivity(first);
    }
    public void NextScreen(View view) {
        Intent next = new Intent(this, NextActivity.class);
        startActivity(next);
    }
    public void DeathScreen(View view) {
        Intent death = new Intent(this, DeathActivity.class);
        startActivity(death);
    }
}

File: app/src/main/java/com/example/kalkulatorbmi/MainActivity.kt
package com.example.kalkulatorbmi

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.example.kalkulatorbmi.SecondeActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun FirstScreen(view: View?) {
        val first = Intent(this, SecondeActivity::class.java)
        startActivity(first)
    }

    fun NextScreen(view: View?) {
        val next = Intent(this, NextActivity::class.java)
        startActivity(next)
    }

    fun DeathScreen(view: View?) {
        val death = Intent(this, DeathActivity::class.java)
        startActivity(death)
    }
}